spec/index.bs

```diff
@@ -1950,6 +1963,20 @@ interface GPUAdapterInfo {
         instead set |adapterInfo|.{{GPUAdapterInfo/description}} to the empty string or a
         reasonable approximation of a description.
 
+    1. If {{GPUFeatureName/"subgroups"}} is supported, set {{GPUAdapterInfo/subgroupMinSize}}
+        to the smallest supported subgroup size. Otherwise, set this value to 4.
```

Comment:
Would Bikeshed permit the use of "subgroup size" in the WebGPU spec to refer to the definition of the term in the WGSL spec?

Reply:
I linked in the description of the field above this point.